pkg/v1/tkg/client/machine_deployment_cc.go

```diff
@@ -332,7 +332,7 @@ func DoDeleteMachineDeploymentCC(clusterClient clusterclient.Client, cluster *ca
 
 	cluster.Spec.Topology.Workers.MachineDeployments = mds
 
-	if err := clusterClient.UpdateResource(cluster, cluster.ClusterName, cluster.Namespace); err != nil {
+	if err := clusterClient.UpdateResource(cluster, cluster.Name, cluster.Namespace); err != nil {
```

Comment:
is this correct @sbueringer  @ykakarap ?

Reply:
Assuming that you want to hand over the name of the cluster - yes.

.ClusterName was always an empty string